VimeoNetworking/Sources/VimeoRequestSerializer.swift

```diff
@@ -196,8 +196,10 @@ final public class VimeoRequestSerializer: AFJSONRequestSerializer
         
         guard let existingUserAgent = request.value(forHTTPHeaderField: Constants.UserAgentKey) else
         {
-            assertionFailure("No existing user agent")
-
```

Comment:
@ghking what if we checked if we are on tvOS before deciding if we should assert or not? So if there are other cases that come up, we'll catch them instead of letting them fall through.